frontend/heart-code/src/components/auth/register-page/RegisterPage.tsx

```diff
@@ -213,7 +213,12 @@ export default function RegisterPage({ onRegister }: Props) {
   function submit() {
     register(registerData)
       .then(onRegister)
-      .catch(() => alert("registration failed"));
+      .catch((error) => showError(error.response.data.detail));
```

Comment:
het staat niet heel duidelijk in de taak, maar volgens mij hadden we afgesproken dat dit niet via een alert maar door een mesage onder op de pagina weer gegeven zou worden als het email adres al in gebruik is.

Reply:
ah , oke, zal even kijken